md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessLogAPI.java

```diff
@@ -76,7 +77,8 @@ RestWrapper list(@RequestParam(value = "page", defaultValue = "0") int startPage
             //s.close();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)